src/Codeception/Module/Yii2.php

```diff
@@ -482,18 +483,19 @@ public function amLoggedInAs($user)
      * ```
      * instead of calling `haveFixtures` in Cest `_before`
      *
-     * @param $fixtures
+     * @param array $fixtures
+     * @param string|null $owner
      * @part fixtures
      */
-    public function haveFixtures($fixtures)
+    public function haveFixtures($fixtures, $owner = null)
     {
         if (empty($fixtures)) {
             return;
         }
-        $fixturesStore = new Yii2Connector\FixturesStore($fixtures);
+        $fixturesStore = new Yii2Connector\FixturesStore($fixtures, $owner);
```

Comment:
As far as I can tell this constructor only takes 1 parameter.

Reply:
Commited FixturesStore